docs/ug/runSh.md

```diff
@@ -27,4 +27,5 @@ You can update the RepoSense command (i.e., the last line) in the `run.sh` to ma
 Depending on which version you wish to use for report generation, add one of the following flags to the line `./get-reposense.py` in `run.sh` (e.g., `./get-reposense.py --release`):
 * `--release`: Use the latest release (Stable)
 * `--master`: Use the latest version of the master branch
-* `--tag TAG` e.g., `--tag v1.6.1`: use the version identified by the git tag given
+* `--tag TAG` (e.g. `--tag v1.6.1`): Use the version identified by the Git tag given
+* `--commit COMMIT` (e.g. `--commit abc123`): Use the version identified by the Git commit SHA given
```

Comment:
Actually, from the perspective of the user, will the user be able to know the difference in behavior of each commit version?

Reply:
I think it really depends on the commits that the user is skipping, but at the bottom of the RepoSense report, there is a link to the commit version that the RepoSense report is built on.

![image](https://user-images.githubusercontent.com/27919917/113133002-c7619700-9251-11eb-8a03-fa2229295d3a.png)